modules/form-field/react/stories/stories_Checkbox.tsx

```diff
@@ -109,6 +125,70 @@ storiesOf('Components|Inputs/Checkbox/React/Left Label', module)
       hintId={hintId}
       labelPosition={FormField.LabelPosition.Left}
     >
-      {control(<Checkbox id="1" label="Checkbox option" />)}
+      {control(<Checkbox label="Checkbox option" />)}
     </FormField>
   ));
+
+storiesOf('Components|Inputs/Checkbox/React/Visual Testing', module)
+  .addParameters({component: Checkbox})
+  .addDecorator(withReadme(README))
+  .add('States', () => (
+    <StaticStates>
+      <Table>
+        <thead>
+          <tr>
+            <th></th>
+            <th>No state</th>
+            <th>Hover</th>
+            <th>Focused</th>
+            <th>Active</th>
+          </tr>
+        </thead>
+        <tbody>
+          {[false, true].map(checked => {
```

Comment:
Should we align on what states we show? For instance, “disabled” is actually a state that is mutually exclusive to “hover” and “focus”. You cannot focus or hover over a disabled checkbox, so we shouldn’t have a visual state that shows an invalid state in reality.